discovery-provider/alembic/versions/2344b086363c_related_artist_index.py

```diff
@@ -0,0 +1,34 @@
+"""related_artist_index
+
+Revision ID: 2344b086363c
+Revises: 0dbe054f29f8
+Create Date: 2022-07-14 17:15:09.641767
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "2344b086363c"
+down_revision = "0dbe054f29f8"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.get_bind().execute(
+        """
+    begin;
+    CREATE INDEX IF NOT EXISTS "related_artists_related_artist_id_idx" ON "related_artists" ("related_artist_user_id", "user_id");
```

Comment:
do we want to drop the existing index? not sure if we need it.

i think this query in `_get_related_artists` could be re-written to be a single aggregate user related artists join to be fast af 8) 

also, mind refreshing the model? https://github.com/AudiusProject/audius-protocol/tree/master/discovery-provider/src/models

Reply:
I looked at query in `_get_related_artists` but think it needs to join against `users` table to pull back the actual user records for the API response... so left as is!